### PR TITLE
BZ1649168 - Further monitoring/metering reference updates.

### PR DESCRIPTION
**Pull request by pneedle-rh**
Applies to branch/enterprise-3.11.

Related to https://bugzilla.redhat.com/show_bug.cgi?id=1649168. This is a second PR related to that BZ, this time with additional changes. The first PR is complete.

Preview available at http://file.fab.redhat.com/~pneedle/optimizing_storage.html.

### CODE REVIEW

**Comment by pneedle-rh** [comment]
@openshift/team-documentation for peer review please. Please can someone also add 'Peer Review Needed', 'branch/enterprise-3.11' labels and 'Next Release' milestone?

**Comment by pneedle-rh** [comment]
@xltian can you please review this from a QE perspective? In particular, can you please confirm whether this paragraph is valid in `scaling_performance/optimizing_storage.adoc`. Is this something we test?:

~~~
 62 You can use {gluster-native} GlusterFS (a hyperconverged or cluster-hosted
 63 storage solution) or {gluster-external} GlusterFS (an externally hosted storage
 64 solution) for block, file, and object storage for {product-title} registry,
 65 logging, and monitoring.
~~~

**Comment by pneedle-rh** [comment]
@s-urbaniak as discussed today can you please review this PR from an SME perspective? Thanks!

File: scaling_performance/optimizing_storage.adoc [hunk 1]
```diff
@@ -150,15 +150,13 @@ not recommended for {product-title} Registry cluster deployment with production
 In an {product-title} hosted monitoring cluster deployment:
 
 * The preferred storage technology is block storage.
+* If you decide to configure file storage, please make sure that it follows POSIX standards.
```
Comment (bergerhoffer) [inline scaling_performance/optimizing_storage.adoc:153]: Don't use "please" (per IBM guidance)

**Review by bergerhoffer — COMMENTED** [review]
Just one minor comment!

File: scaling_performance/optimizing_storage.adoc (hunk 1 above)
Comment (pneedle-rh) [reply scaling_performance/optimizing_storage.adoc]: Fixed.

**Comment by xltian** [comment]
@juzhao @anpingli @wzheng1  Can you please review? Thanks

**Comment by juzhao** [comment]
file https://github.com/openshift/openshift-docs/blob/46beafc2eed2f05c208d846a337d20958247c50b/scaling_performance/optimizing_storage.adoc
from this PR
"Table 2. Recommended and configurable storage technology", there is "ROX" and "RWX" in the table, but there is not "RWO" abbreviation for ReadWriteOnce, any reason we don't need to mention it?

**Comment by juzhao** [comment]
from file, https://github.com/openshift/openshift-docs/blob/46beafc2eed2f05c208d846a337d20958247c50b/scaling_performance/optimizing_storage.adoc
Table 2. Recommended and configurable storage technology

RWO which is abbreviation for ReadWriteOnce, are we intended not to mention it?

**Comment by pneedle-rh** [comment]
> from file, https://github.com/openshift/openshift-docs/blob/46beafc2eed2f05c208d846a337d20958247c50b/scaling_performance/optimizing_storage.adoc
> Table 2. Recommended and configurable storage technology
> 
> we see ROX and RWX, but RWO which is abbreviation for ReadWriteOnce is missing, are we intended not to mention it?

@s-urbaniak @sichvoge can you please advise on this point? Should we add an RWO column to table 2 and if so, what are our recommendations in context to that for block, file and object storage? Thanks!

**Comment by s-urbaniak** [comment]
lgtm but I'd love to have another pair of eyes from the monitoring team.
/cc @openshift/openshift-team-monitoring

**Comment by juzhao** [comment]
lgtm for the monitoring part, waiting for others to review their parts

**Review by lilic — COMMENTED** [review]
👍 lgtm

**Comment by juzhao** [comment]
> from file, https://github.com/openshift/openshift-docs/blob/46beafc2eed2f05c208d846a337d20958247c50b/scaling_performance/optimizing_storage.adoc
> Table 2. Recommended and configurable storage technology
> 
> we see ROX and RWX, but RWO which is abbreviation for ReadWriteOnce is missing, are we intended not to mention it?

split to bug https://bugzilla.redhat.com/show_bug.cgi?id=1829650 to not affect this PR get merged

**Comment by pneedle-rh** [comment]
> > from file, https://github.com/openshift/openshift-docs/blob/46beafc2eed2f05c208d846a337d20958247c50b/scaling_performance/optimizing_storage.adoc
> > Table 2. Recommended and configurable storage technology
> > we see ROX and RWX, but RWO which is abbreviation for ReadWriteOnce is missing, are we intended not to mention it?
> 
> @s-urbaniak @sichvoge can you please advise on this point? Should we add an RWO column to table 2 and if so, what are our recommendations in context to that for block, file and object storage? Thanks!

This additional requirement has now been forked to new Bugzilla https://bugzilla.redhat.com/show_bug.cgi?id=1829650 and so is beyond the scope of this current PR. I will follow up on this point in that new Bugzilla. Thank you @juzhao for creating that new BZ!

**Comment by pneedle-rh** [comment]
@anpingli @wzheng1 is there any further QE review required? Thanks!

**Review by anpingli — COMMENTED** [review]
LGTM

**Comment by wzheng1** [comment]
I didn't see any change to registry. So anping and juzhao's ack is enough for this doc change.

**Comment by pneedle-rh** [comment]
@openshift/team-documentation can someone please merge this? Thank you!

**Pull request merged**